README.md

```diff
@@ -42,6 +42,26 @@ Include `mh-opsworks-recipes` in your node's `run_list`:
 
 * Dan Collis-Puro - [djcp](https://github.com/djcp)
 
+## Development
+
+* Create a branch off the develop branch
+* Do your work and test it in a running mh-opsworks cluster
+* Be sure to update the "To be released" section of the CHANGELOG.md file with
+  a description of your change, and whether or not it requires a manual recipe
+  invocation. If your change runs as part of the deploy lifecycle, it will be run
+  the next time the main app is deployed to a cluster and therefore does not
+  require a manual chef recipe invocation.
```

Comment:
As I'm about to try following this process I realize that a tip on how to quickly determine this would be helpful.